### DISCUSSION TITLE
Unable to find Renovate-Branch

### DISCUSSION THREAD

**Discussion by Zusel** [renovatebot/renovate#31409] [Request Help]
### What would you like help with?

Other

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Github Enterprise, Renovate-Docker(37.422.1)

### Please tell us more about your question or problem

Hey,
we were wondering why no PR has been created for an update yet. While checking the debug logs, we noticed that Renovate found the update and even indicated that a branch had been created. Unfortunately, this branch does not exist. How is this possible? Thank you very much.
![renovate](https://github.com/user-attachments/assets/6c1deebd-d3eb-43fd-bd6b-140c7ab8a658)

**Comment by Zusel** [comment]
edit: same with Version: 38.56.0

**Comment by rarkins** [comment]
> and even indicated that a branch had been created

This is a misunderstanding. The field you highlighted doesn't mean it's been created, just that this would be the name if it was.

You need to search your logs for other hits for this branch name

**Comment by Zusel** [comment]
Yes, there are 2 more JSON objects with the same entry (and the same version) that look identical. There is also a PR from the previous version that was closed, but the branch still exists. So, this version should be skipped.

**Reply by rarkins** [reply]
This original statement:

> we were wondering why no PR has been created for an update yet

and your recent statement:

> this version should be skipped

seem to contradict each other.

Also, I guarantee you that the branch name shows up more than two more times if you've captured the full debug logs.

**Reply by Zusel** [reply]
Now, let's go into more detail: We want to use vue-i18n version 10.X.X, and version 9.14.0 should be skipped. Shortly after I took the screenshot, we rolled back to version 9.13.X because 9.14.0 is broken. That's why it should be skipped. Now the question is, why isn't a PR being created for version 10.X.X even though the log shows that it recognized/found the version?

**Reply by Zusel** [reply]
> Also, I guarantee you that the branch name shows up more than two more times if you've captured the full debug logs.
![image](https://github.com/user-attachments/assets/bbfbdec2-e3d4-4afe-a736-a650204f42d9)

3 Times total

**Reply by rarkins** [reply]
Please provide the full debug logs

**Reply by Zusel** [reply]
Hey, thanks for the quick responses. Unfortunately, I can't provide the logs. I understand that this is inconvenient for debugging. Sorry.
